payjoin/src/send/multiparty/mod.rs

```diff
@@ -22,18 +22,57 @@ mod error;
 pub struct SenderBuilder<'a>(v2::SenderBuilder<'a>);
 
 impl<'a> SenderBuilder<'a> {
-    pub fn new(psbt: Psbt, uri: PjUri<'a>) -> Self { Self(v2::SenderBuilder::new(psbt, uri)) }
-    pub fn build_recommended(self, min_fee_rate: FeeRate) -> Result<Sender, BuildSenderError> {
-        let v2 = v2::SenderBuilder::new(self.0 .0.psbt, self.0 .0.uri)
-            .build_recommended(min_fee_rate)?;
-        Ok(Sender(v2))
+    pub fn new(psbt: Psbt, uri: PjUri<'a>) -> Self {
+        Self(crate::send::v2::SenderBuilder::new(psbt, uri))
```

Comment:
Not sure why this was changed to be fully qualified

Reply:
Not sure either. Maybe something automated by my ide ? Fixed